version/couchdb_2_1_1.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash -e
-echo "================= Installing Cassandra 2.1.1 ==================="
+echo "================= Installing Couchdb 2.1.1 ==================="
 echo "[bintray--apache-couchdb-rpm]
 name=bintray--apache-couchdb-rpm
 baseurl=http://apache.bintray.com/couchdb-rpm/el7/x86_64/
 gpgcheck=0
 repo_gpgcheck=0
 enabled=1" > /etc/yum.repos.d/bintray-apache-couchdb-rpm.repo
+# couchbd is failing if its on this path so moving it to other location 
+# https://www.spinics.net/lists/centos/msg169867.html
+mv /usr/lib/python2.7/site-packages/urllib3/packages/ssl_match_hostname  /usr/lib/python2.7/site-packages/urllib3/packages/ssl_match_hostname.backup
```

Comment:
is urllib3 working after this change? we need to make sure that whatever package was installing it is working correctly

Reply:
ya its working fine now  its printing correct version

![image](https://user-images.githubusercontent.com/37651858/40229680-987e40ac-5ab2-11e8-8ac5-b6037d64d89d.png)

Reply:
im asking is urllib3 working, not couchdb. we're changing the files in urllib3 and there are packages like apachelibcloud that depend on it. how are we verifiying that those are working correctly?

Reply:
i used the script which uses urllib3 in apachelibcloud .this is working fine

![image](https://user-images.githubusercontent.com/37651858/40231773-f9c92b04-5ab9-11e8-8dd6-354ebf6eae4d.png)

Reply:
ok. looks good